### DISCUSSION TITLE
Getting started with dumping unknown games

### DISCUSSION THREAD

**Discussion by PsychoFox11** [sanni/cartreader#759] [General Chat]
I have quite a few carts, from 30 year old multicarts, to a ton of recent releases, that I'd love to get the hash data etc. from, but aren't recognized by the reader. For ones where there isn't much info online about it, how do you go about finding the size, mapper, etc.? Or in the case of NES the mapper, and 2 different sizes. I have a 52-in-1 NES cart that I can find the mapper for, but have no idea about its sizes. Do I just need to wait for someone smarter to figure them out?

**Comment by Ancyker** [comment]
Take them apart and look pretty much.

**Reply by PsychoFox11** [reply]
I guess I need to learn how to know what I'm looking at then! :)

**Reply by PsychoFox11** [reply]
Trying to figure out what this means for now lol.
![image](https://user-images.githubusercontent.com/5882157/229961702-5474c7b0-ae7f-4e85-808d-08ab28b68c51.png)

**Reply by Ancyker** [reply]
those are 3x 4Mb (512KB) chips

**Reply by PsychoFox11** [reply]
Did you just search the chip numbers? How do I know how much CHR and how much PRG? I do know the mapper for this one, but can you tell from looking?
I appreciate the info/help, and don't expect you to teach me everything, but am interesting in contributing what I can to no-intro. I've moved dozens of roms to verified status there, would like to contribute new stuff.

**Reply by Ancyker** [reply]
I just searched the chip model.

**Reply by Cephiros** [reply]
What is the mapper? If you cant figure it out use trial and error. Gotta be either 512KB prg, 1MB chr or 1MBprg, 512KB chr, or 1.5MB prg
